shipyard-controller/docs/docs.go

```diff
@@ -1,13 +1,14 @@
-// Package docs GENERATED BY THE COMMAND ABOVE; DO NOT EDIT
+// GENERATED BY THE COMMAND ABOVE; DO NOT EDIT
 // This file was generated by swaggo/swag
+
 package docs
 
 import (
 	"bytes"
 	"encoding/json"
 	"strings"
-	"text/template"
 
+	"github.com/alecthomas/template"
```

Comment:
Where is this coming from? This is introducing a new dependency in `go.mod`

Reply:
this is required by the swagger doc

Reply:
It's not maintained though.
![image](https://user-images.githubusercontent.com/56065213/117319960-14a5d980-ae8c-11eb-86d4-af77de275230.png)

Reply:
i have removed it